plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/job/PodLogsJob.java

```diff
@@ -158,26 +158,34 @@ public void run() {
 		protected void runOCBinary(MultiStatus multiStatus) {
 			ConsoleUtils.registerConsoleListener(new ConsoleListener(this));
 			final MessageConsole console = showConsole();
-			final MessageConsoleStream stream = console.newMessageStream();
+			final MessageConsoleStream os = console.newMessageStream();
+			os.setEncoding("UTF-8");
+			BufferedOutputStream stream = new BufferedOutputStream(os);
```

Comment:
I wonder if we need the buffer here. Below, data is read by big enough portions up to 1024 bytes, which is by itself a good buffer. If data is stored in the buffer output stream until the limit (8192) is reached, os can be closed before any output is shown. I cannot be sure, at least it is worth testing with/without buffer, whatever behaves nicer (and if no difference is noticed, the buffer is not needed too).